src/Uno.UI.RemoteControl.Host/IDEChannel/IdeChannelServer.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Threading.Tasks;
+using Uno.UI.RemoteControl.Messaging.IdeChannel;
+
+namespace Uno.UI.RemoteControl.Host.IdeChannel;
+
+internal class IdeChannelServer : IIdeChannelServer
+{
+	private IServiceProvider _serviceProvider;
```

Comment:
This is not used ... and usually, unless specific cases, I think that we should request dependencies in the constructor instead of resolving lazily at runtime.